### PR TITLE
Fixing issue 18: changing default fallback behaviour when chainId is undefined

### PR DESCRIPTION
**Pull request by NindoK**
This PR solves the issue when chainId initially is undefined. By using the default 
`const chainString = chainId ? parseInt(chainId).toString() : "31337"` we might incur in an error on the sequent line if networkMapping doesn't have a value for `31337`.

With this PR we just wait until useMoralis() retrieve the chainId and if it's able to do so, it will update the page with the correct data avoiding any issue.

### CODE REVIEW

**Comment by PatrickAlphaC** [comment]
Oh awesome!

Could you add a ternary operator for the `NFTBox` to not render if the `marketplaceAddress` isn't defined?

**Comment by NindoK** [comment]
@PatrickAlphaC should be done. Let me know if this was what you intended! :)

File: pages/index.js [hunk 1]
```diff
@@ -32,6 +31,8 @@ export default function Home() {
                                     seller={seller}
                                     key={`${nftAddress}${tokenId}`}
                                 />
+                            ) : (
+                                <div></div>
```
Comment (PatrickAlphaC) [inline pages/index.js:35]:
```suggestion
                                <div>Network error, please switch to a supported network. </div>
```

**Comment by PatrickAlphaC** [comment]
Thank you!

**Comment by NindoK** [comment]
Thank you!

**Pull request merged**